source/docs/prismic/resolver-cache.md

```diff
@@ -0,0 +1,164 @@
+---
+id: resolver-cache
+title: Cache Prismic Resolvers
+---
+
+The `PrismicCachedResolver` is a decorator for a resolver that returns Prismic Content, the resolver allows us to serialize and deserialize the cached data with the [defineContentTransformers](/docs/prismic/expose-content.html#defineContentTransformers-typeIdentifier-options).
+
+In order to create references from prismic to the cached documents, we need to ensure that the `PrismicCachedResolver` receives a [Content](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/main/prismic/server/modules/prismic/core/loaders/Content.js) type from your resolver. The resolver can either cache a `singleton` Content, or a `list` of Content.
+
+## Resolver Cache API
+
+#### `PrismicCachedResolver(resolver[, options])`
+
+Returns a resolver that caches the result of the given resolver.
+
+**Arguments:**
+
+- `resolver`: A [custom resolver](/docs/advanced/graphql/change-resolver-behavior.html#Implement-your-custom-resolver-logic) that returns a `Content` or an array of `Content`,
+- `options` : An object with the following properties:
+  - `mapParamsToId` A function that maps the resolver parameters to a unique id.
+  - `contentProperty` The name of the property that contains the [Content](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/main/prismic/server/modules/prismic/core/loaders/Content.js).
+
+## Usage
+
+To use the `PrismicCachedResolver` you need to import it into your `resolvers.js` and wrap your resolver with decorator.
+
+```js
+// my-module/resolvers.js
+import PrismicCachedResolver from "prismic/server/modules/prismic/core/cache/PrismicCachedResolver";
+
+export default {
+  Query: {
+    resolver: PrismicCachedResolver((_, __, { loaders }) =>
+      loaders.MyModule.loadAll()
+    ),
+  },
+};
+```
+
+### Singleton Content
+
+This is the most simple usecase. The `PrismicCachedResolver` will cache a singleton Content.
+
+So let's say we want to cache a resolver that loads a single Faq entry, we can simple decorate our original resolver
+
+```js
+import PrismicCachedResolver from "prismic/server/modules/prismic/core/cache/PrismicCachedResolver";
+
+export default {
+  Query: {
+    faqEntry: PrismicCachedResolver((source, { slug }, { loaders }) => {
+      return loaders.FaqLoader.loadBySlug(slug); // returns Promise<Content>
+    }),
+  },
+};
+```
+
+But now the issue is, when we call **faqEntry** resolver, it will always load the same cached faq entry, to avoid this we need map the `slug` parameter to the cache key, you can pass in the `mapParamsToId` option to the `PrismicCachedResolver` to do this.
```

Comment:
```suggestion
Now the **faqEntry** resolver will always load the same cached faq entry.

To avoid this issue, map the `slug` parameter to the cache key, by passing the `mapParamsToId` option to the `PrismicCachedResolver`.
```